airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_streams.py

```diff
@@ -55,12 +55,20 @@ def __init__(
         self.page_size = page_size if page_size is not None else 100
         self._filter_statuses = filter_statuses
         self._fields = None
+        self._saved_fields = None
```

Comment:
nit: let's call the field `cached_selected_fields` so we're clear on what saved means.

An alternative could also be to add a `@cache` annotation to the method
```
@cache
def fields(self, **kwargs):
   ...
```

Reply:
I like the cache approach, will add this to base streams.